sqlitecluster/SQLiteNode.cpp

```diff
@@ -1278,7 +1278,19 @@ void SQLiteNode::_onMESSAGE(SQLitePeer* peer, const SData& message) {
             peer->commandAddress = message["commandAddress"];
         }
         peer->setCommit(message.calcU64("CommitCount"), message["Hash"]);
-
+        
+        // We check the commit difference with 12,500 commits behind because that 
```

Comment:
Is this change supposed to be in this PR? Seems kind of unrelated to the other change.

Reply:
I think this PR should be on hold til the other one is merged

Reply:
this was the other one: https://github.com/Expensify/Bedrock/pull/2039

I changed the base branch to that PR so you can see only the necessary changes.